motr/client.h

```diff
@@ -632,7 +647,10 @@ enum m0_entity_type {
 	M0_ENF_META = 1 << 0,
 	/**
 	 * If this flags is set during entity_create() that means application
-	 * do not support update operation. This flag is not in use yet.
+	 * do not support update operation.
```

Comment:
There was also RMW being triggered for fully spanned parity groups as well.
Can you check if this is still the case ?

Reply:
I didn't see this happening for the fully spanned groups. Sounds like a bug. Any idea on how to reproduce it?

Reply:
May be i haven't checked the io size for each PG sent by S3 layer(RGW SAL), the case mentioned in [PR-288](https://github.com/Seagate/cortx-rgw/pull/288) might be occurring, which triggered RMW for non-last parity groups.